### PR TITLE
implement the dropdown to select the dataset to load/display

### PR DESCRIPTION
**Pull request by Salam-Dalloul**
all requirements of [CELE-63](https://metacell.atlassian.net/browse/CELE-63) [CELE-64](https://metacell.atlassian.net/browse/CELE-64) have been pushed to this [PR](https://github.com/MetaCell/c-elegans-app/pull/8) and here is the rest of the implementation 
-  retrieve all the datasets loaded already from the context? if yes retrieve these to display them in the dropdown.
- create a common Autocomplete component to be used in the compare dialog and the 3D viewer
-  retrieve all the neurones to replace the dummy data inside of the compare dialog

[CELE-63]: https://metacell.atlassian.net/browse/CELE-63?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CELE-64]: https://metacell.atlassian.net/browse/CELE-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

https://github.com/MetaCell/c-elegans-app/assets/32265731/67ded575-10cd-44c3-b013-7a60a0b7721e

### CODE REVIEW

**Review by ddelpiano — CHANGES_REQUESTED** [review]
@Salam-Dalloul some icons are missing in the 3d viewer, even if the functionalities are not ready yet the icons should be implemented. Thanks

**Review by aranega — APPROVED** [review]
Looks good to me :) Thanks @Salam-Dalloul

File: applications/visualizer/frontend/src/models/workspace.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class Workspace {
 
     store: ReturnType<typeof configureStore>;
     layoutManager: any;
-
+    datasets: Array<Dataset>;
```
Comment (afonsobspinto) [inline applications/visualizer/frontend/src/models/workspace.ts]: question (non-blocking): Do we need this here?

**Review by afonsobspinto — APPROVED** [review]

File: applications/visualizer/frontend/src/models/workspace.ts (hunk 1 above)
Comment (aranega) [reply applications/visualizer/frontend/src/models/workspace.ts]: I guess we will need to have the datasets that are attached to a workspace for the comparison mode later no?

Comment (afonsobspinto) [reply applications/visualizer/frontend/src/models/workspace.ts]: I spoke with @Salam-Dalloul about this. My understanding is that the property I highlighted can be removed. Initially she though the workspace would be a good place to have the information about all datasets but after some though she decided to have them in the global context (but forgot to remove this bit).
To know what datasets are associated to a workspace we have the activeDatasets property.

File: applications/visualizer/frontend/src/components/viewers/ThreeD/ThreeDViewer.tsx [hunk 2]
```diff
@@ -36,6 +38,7 @@ function ThreeDViewer() {
     const [showSynapses, setShowSynapses] = useState<boolean>(true);
     const [instances, setInstances] = useState<Instance[]>([])
     const [isWireframe, setIsWireframe] = useState<boolean>(false)
+    const { datasets} = useGlobalContext();
```
Comment (afonsobspinto) [inline applications/visualizer/frontend/src/components/viewers/ThreeD/ThreeDViewer.tsx]: question (non-blocking): Shouldn't this be the active datasets (the ones that are active in the current workspace), opposed to all the existent datasets?

**Pull request merged**